tests/deployment/switch_config.py

```diff
@@ -54,7 +61,30 @@ def server_init():
                                      'site_layout')
 
 
-class TestNativeNetwork(NetworkTest):
+@pytest.fixture
+def not_dell():
+    """open the site-layout file to see if we don't have a dell switch"""
+
+    layout_data = open('site-layout.json')
+    layout = json.load(layout_data)
+    layout_data.close()
+    for switch in layout['switches']:
+        return switch['type'] != DELL
```

Comment:
This is only going to check the first switch. If this was your intent, you can dispense with the loop and just check the first element directly. Otherwise, you've got a bug.